guides/waf-waiver.md

```diff
@@ -0,0 +1,19 @@
+# WAF Waivers
+Azure Web Application Firewall (WAF) on Azure Application Gateway provides centralized protection of your web applications from common exploits and vulnerabilities. Web applications are increasingly targeted by malicious attacks that exploit commonly known vulnerabilities. SQL injection and cross-site scripting are among the most common attacks.
+
+WAF on Application Gateway is based on Core Rule Set (CRS) 3.1, 3.0, or 2.2.9 from the Open Web Application Security Project (OWASP). The WAF automatically updates to include protection against new vulnerabilities, with no additional configuration needed.
+
+All of the WAF features listed below exist inside of a WAF Policy. You can create multiple policies, and they can be associated with an Application Gateway, to individual listeners, or to path-based routing rules on an Application Gateway. This way, you can have separate policies for each site behind your Application Gateway if needed.
+
+If a waiver is required for any policy then this must be approved by Security and implemented by CSC.
```

Comment:
I know there is a bit more context below about having to forward the waiver to security but I wouldn't know who it what `Security` are. Is it possible to add some more context here (in this doc) or is it best to have people ask that question when a waiver is required?

Reply:
I thought the same when writing it but it's difficult to give more context as who will be approving them in what forum may be subjective.  At the moment we're just raising direct with our security architect but that may not be practical long term. I think we can refine this as we learn what works at scale.